### DISCUSSION TITLE
Droppable gear: single ally stagger cure

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#353] [Ideas]
Upgrades: Cleansing, Agile, Guarding

**Comment by doubledh** [comment]
can... you cleanse stagger after it hits the cap but before the turn end?

Does that make this counter to the support role trying to speed up?

**Reply by ntseng** [reply]
Stagger is only checked at end of round, so relieving stagger that would be promoted to stun before end of round to avoid stun is possible. Because "teetering on the brink" is exciting and "prevent difference in kind" is more visible, "right before an ally would be stunned" is probably a relatively fun timing to do that.

If I'm understanding correctly, the antergy case you're highlighting is "support can't relieve stagger on an ally because support is going 'first' and ally is at 0 stagger at the beginning of the round, but ally will get stunned by the end of the round". If this turns out to be the case frequently, I think it's more appropriate to buff Poise or make Stagger less bursty since stagger relief is the counterplay mechanic in this system.

Given those two considerations, we might even be slightly leaning toward encouraging players to do stagger relief right before stun. However I think this timing is strategically more constrained (either it's a niche strategy or the player must act fully reactively [removing choice]) and, having not noticed the timing window, we haven't planned power budget or UI affordance for adding an incentive to using the "right before stun" timing, maybe an addition to consider later when we're trying to add more depth to the game.
